metrics_test.go

```diff
@@ -559,3 +560,313 @@ func TestMetricsToMonitoringMetrics_fromProtoPoint(t *testing.T) {
 		}
 	}
 }
+
+func TestGetMonitorResource(t *testing.T) {
```

Comment:
nit: Should the name be `TestResourceByDescriptor`?

Reply:
changed.